unified-codes/apps/data-service/src/app/v2/types/entity/ProductEntity.ts

```diff
@@ -0,0 +1,42 @@
+import { EEntityType, IEntity, Entity } from './Entity';
+import { IRouteEntity, RouteEntity } from './RouteEntity';
+import { IProperty, Property } from '../Property/Property';
+
+export enum EProductField {
```

Comment:
the enum isn't used anywhere, can it be removed?

Reply:
Few things in `v2` which aren't currently used (as the resolvers and schema isn't implemented yet). Thinking these enums will probably be needed for the graph definitions (but if not, can remove them then!).